mlx/primitives.cpp

```diff
@@ -2130,10 +2130,11 @@ std::vector<array> Scatter::vjp(
     case Scatter::None:
     case Scatter::Sum:
     case Scatter::Max:
+    case Scatter::Min:
       break;
     default:
       throw std::runtime_error(
-          "[scatter] VJP implemented only for scatter and scatter_add");
+          "[scatter] VJP not implemented for this scattering reduction type");
```

Comment:
Nitpick but I would change this to "VJP not implemented for scatter_prod" since it is the only op left.